eclipse-che/che-server
# Simplify the Raw devfile filename check

### What does this PR do?
Do not check the Raw devfile file name but check if the url ends with `.yaml` before using the Raw devfile url resolver.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/22368

### How to test this PR?


1. Deploy che with the PR image: `quay.io/eclipse/che-server:pr-548`
2. In a public GitHub repository create a `custom.yaml` file with a simple devfile content:
```YAML
schemaVersion: 2.2.0
metadata:
  name: test
```
3. Copy the `Raw` file location.
4. [Start a workspace from the raw devfile location](https://eclipse.dev/che/docs/stable/end-user-guide/starting-a-new-workspace-from-a-devfile-url/)

See: workspace starts from the devfile.
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
